Change wording for rollover and index (#3080)
opensearch-project/documentation-website: _dashboards/im-dashboards/rollover.md

---
layout: default

> title: Roll over

Reviewer: This is a noun, so should be one word.
```suggestion
title: Rollover
```

parent: Index management in Dashboards
nav_order: 40
redirect_from:
  - /dashboards/admin-ui-index/rollover/
---

> # Roll over

Reviewer: Same
```suggestion
# Rollover
```

Introduced 2.6
{: .label .label-purple }

> OpenSearch Dashboards allows you to perform an [index roll over]({{site.url}}{{site.baseurl}}/im-plugin/ism/error-prevention/index/#rollover) operation with **Index Management**.

Reviewer 2: Should we explain why someone would want to do this?

Reviewer 2: "with Index Management"? How about, "You can perform a roll over operation in OpenSearch Dashboards when you want to ...."

Author: This is explained in the ISM/data-stream/API documentation as to the “why” you would do it. As this is a front-end for an existing feature I wouldn’t think so. Also, I have linked to that content in this page.

> OpenSearch Dashboards allows you to perform an [index roll over]({{site.url}}{{site.baseurl}}/im-plugin/ism/error-prevention/index/#rollover) operation with **Index Management**.

Reviewer: Same: noun
```suggestion
OpenSearch Dashboards allows you to perform an [index rollover]({{site.url}}{{site.baseurl}}/im-plugin/ism/error-prevention/index/#rollover) operation with **Index Management**.
```

## Data streams

> To perform a roll over operation on a data stream, perform the following steps:

Reviewer: Adjective
```suggestion
To perform a rollover operation on a data stream, perform the following steps:
```

1. Under **Index Management**, choose **Data streams**.

1. Choose **Actions**, and then choose **Roll over**, as shown in the following image.

    ![Roll over]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover1.png)

1. Under **Configure source**, select the source data stream on which you want to perform the rollover operation.

1. Choose **Roll over**, as shown in the following image.

    ![Roll over]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover3.png)

## Aliases

> To perform a roll over operation on an alias, perform the following steps:

Reviewer: ```suggestion
To perform a rollover operation on an alias, perform the following steps:
```

1. Under **Index Management**, choose **Aliases**.

1. Choose **Actions**, and then choose **Roll over**, as shown in the following image.

    ![Roll over]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover2.png)

> 1. Under **Configure source**, select the source alias on which you want to perform the roll over operation.

Reviewer: ```suggestion
1. Under **Configure source**, select the source alias on which you want to perform the rollover operation.
```

> 1. If the alias does not contain a write index, you will be prompted to assign a write index, as shown in the following image. 

Reviewer 2: you _are_ prompted

Author: 👍

    ![Roll over]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover4.png)

> 1. Under **Configure a new roll over index** and on the **Define index** pane, specify an index name and an optional index alias.

Reviewer: ```suggestion
1. Under **Configure a new rollover index** and on the **Define index** pane, specify an index name and an optional index alias.
```

1. Under **Index Management**, choose **Data streams**.

1. Choose **Actions**, and then choose **Roll over**, as shown in the following image.

    ![Roll over]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover1.png)

1. Under **Configure source**, select the source data stream on which you want to perform the rollover operation.

1. Choose **Roll over**, as shown in the following image.

    ![Roll over]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover3.png)

## Aliases

To perform a rollover operation on an alias, perform the following steps:

1. Under **Index Management**, choose **Aliases**.

1. Choose **Actions**, and then choose **Roll over**, as shown in the following image.

    ![Roll over]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover2.png)

1. Under **Configure source**, select the source alias on which you want to perform the rollover operation.

> 1. If the alias does not contain a write index, you will be prompted to assign a write index, as shown in the following image. 

Author: ```suggestion
1. If the alias does not contain a write index, you are prompted to assign a write index, as shown in the following image. 
```
